src/test/java/HW20_Lombok/HeroBuilderTest.java

```diff
@@ -0,0 +1,21 @@
+package HW20_Lombok;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class HeroBuilderTest {
+
+    @Test
+    void shouldGet(){
+
+        var hero = HeroBuilder.builder()
+                .name("Abraxas")
+                .gender("Male")
+                .eyeColor("blue");
```

Comment:
```suggestion
                .eyeColor("blue")
                .build();
```
 а далі build вже не треба використовувати.